scripts/release-train-issue-template.md

```diff
@@ -41,7 +41,8 @@ Wind down PR queue. There has to be enough time after the last (non-trivial) PR
 ### Cutting the release
 
 - [ ] Make sure there are no stray staging repos on sonatype
-- [ ] Wait until [master build finished](https://travis-ci.org/akka/akka-http/builds/) after merging the release notes
+- [ ] Wait until [master build finished](https://travis-ci.org/akka/akka-http/builds/) after merging the release notes (otherwise, the master
+      build might pick up the tag and start publishing the release uhoh)
```

Comment:
```suggestion
      build might pick up the tag and start publishing the release uhoh -- So that Johannes doesn't believe he can be faster by ignoring this step.)
```
;-)

Reply:
Somehow I had to make up for the time lost by the broken akka master nightlies...